code/modules/projectiles/updated_projectiles/projectile.dm

```diff
@@ -455,6 +459,11 @@
 	if(P.ammo.debilitate && stat != DEAD && ( damage || (P.ammo.flags_ammo_behavior & AMMO_IGNORE_RESIST) ) )
 		apply_effects(arglist(P.ammo.debilitate))
 
+	if(P.ammo.reagent_type && stat != DEAD && P.reagent_amount)
+		if(istype(src, /mob/living/carbon/human))
+			var/mob/living/carbon/human/H = src
+			H.reagents.add_reagent(P.ammo.reagent_type, max(0,P.reagent_amount))
+
```

Comment:
```
if(P.list_reagents && stat != DEAD)
	if(ishuman() || ismonkey())
		reagents.reaction(M, TOUCH) // potential extra effects
		P.reagents.trans_to(src, 50)
```

Reply:
My apologises, I asininely copy pasted the reagents.reaction line, it should be `P.reagents.reaction(src, TOUCH)`

Reply:
Yeah, I figured; I already made that change.

Reply:
Yes, but also the `P.` before the datum proc, because you are setting up the reaction for the projectile's chemicals, not the mob's.